app/views/calculators/calculator.html.slim

```diff
@@ -49,9 +49,9 @@
     .col-sm.result-card
       .img-border
         = image_pack_tag "diaper_new.png", class:"rounded"
-      span  
+      span
         p data-type="to_be_used_diapers_amount" 0
-        = t '.will_buy_diapers'
+        div id="localized_diaper_string" Підгузки, які будуть використані
```

Comment:
Make it to work with english localization as well

Reply:
fixed